### PR TITLE
Add the cart page

### PR DESCRIPTION
**Pull request by rainerdema**
Quick Info
---

| Issue |
| -- | 
| #25  |

1. Adds the checkout cart page
2. Adds the checkout cart page link from the index page
3. `Don't allow the use of caches` except in production: The use of caches in non-production environments is no longer allowed to avoid problems during the development.
4. Fix `Storybook` with compatibility updates for `React 17.x`
Screenshot 2020-10-30 at 17 56 08
5. Fix `ApolloClient` initialization without [`OrderToken`](https://github.com/nebulab/solidus_nextjs_frontend/blob/rainerd/add-cart-page/lib/apolloClient.js#L29-L35)
6. Align the `ApolloClient` configuration with the changes in the [official example on `next.js`](https://github.com/vercel/next.js/blob/canary/examples/with-apollo/lib/apolloClient.js)

### Screenshots
Screenshot 2020-10-30 at 18 51 55

### Note
This page will need other features:
1. Add / Remove quantities of products from the cart
2. Remove products from the cart

### CODE REVIEW

File: containers/line-items/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@apollo/client'
```
Comment (dinohamzic) [inline containers/line-items/index.js:1]: I don't think we are using this whole component at the moment anywhere. Should we delete it?

File: pages/cart.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    return <div>Loading</div>
+  }
+
+  const { lineItems } = data.currentOrder
+
+  return (
+    <main>
+      <p>Shopping Cart</p>
+      <ul>
+        {lineItems.nodes.map(line_item => (
```
Comment (dinohamzic) [inline pages/cart.js:21]: We need an empty state for the cart (when `lineItems.nodes` is empty) like you did in `containers/line-items/index.js`.

**Review by dinohamzic — CHANGES_REQUESTED** [review]

**Review by vassalloandrea — APPROVED** [review]
Great 🔥

**Review by dinohamzic — APPROVED** [review]
Good job!

**Pull request merged**